### PR TITLE
Updated Czech translation

### PR DESCRIPTION
**Pull request by buresdv**
Updated the Czech translation to cover all new strings.

I have found two minor issues that I couldn't manage to fix in the "Love" pane:
Snímek obrazovky 2020-11-22 v 1 34 40

I couldn't find the source string for "Tip me a...", so I couldn't translate it, and the bottom text is a bit misaligned because the translation for "Privacy Policy" is a bit longer than the original :(

### CODE REVIEW

**Comment by aonez** [comment]
Thanks a lot @buresdv! 

Will check that misalign first. The tips strings come directly from Apple's servers, they can be translated there. But maybe I can add them anyway locally and translate them in-app. Let me check.

**Review by aonez — CHANGES_REQUESTED** [review]
You should restore Translations/README.md and Translations/convert files before I can merge this one

File: Translations/cs.lproj/Credits.strings
Comment (aonez) [inline Translations/cs.lproj/Credits.strings:11]: I'm not sure changing Keka, being it the product name, is a good idea. Is this a common practice in Apple's macOS bundled apps?

**Review by aonez — CHANGES_REQUESTED** [review]

File: Translations/cs.lproj/Credits.strings
Comment (buresdv) [reply Translations/cs.lproj/Credits.strings:11]: I looked into the Czech translation memory for macOS and they use declension like this when the resulting word isn't too different from the original. I think that in this case, it's obvious what this changed word is referring to. This is why I didn't change it in occurrences where the final word would have more than one letter different (e.g. "Kece").

I think that in this case, the changed word helps the text flow better, but it's no big deal, if you think it shouldn't be changed, it doesn't have to be 👍

**Comment by buresdv** [comment]
Will do in a few when I get to my computer, I don't know how to restore from a phone.

**Comment by buresdv** [comment]
Hey, I need some help with restoring the two files. I'm not exactly sure how to. As you can see, I added another commit to this PR, attempting to reverse the two files to a previous commit using _checkout_, but I don't think that's what you had in mind.

Could you please tell me what to do? I guess I'm not yet skilled enough with git 😅

**Comment by aonez** [comment]
David you did it! 😉

**Comment by buresdv** [comment]
Yaaay 😆

File: Translations/cs.lproj/Credits.strings
Comment (aonez) [reply Translations/cs.lproj/Credits.strings:11]: So forgive my lack of knowledge but you will use those declensions in own names?

Like: `Děkujeme, že používáte Davidu!`

Comment (buresdv) [reply Translations/cs.lproj/Credits.strings:11]: It's okay, Czech is extremely complex 😦 Yes, every single noun, including names, has seven possible declensions. In the case of David, this sentence would be Děkujeme, že používáte Davida! (because Keka is feminine and David is masculine).

I accidentally somehow managed to start a review, I'm sorry for that, Github is still very confusing to me 😅

**Review by aonez — APPROVED** [review]

**Comment by aonez** [comment]
Let's go like this! Thanks a lot David!

Tomorrow I'll add the tips strings :)

**Comment by aonez** [comment]
@buresdv already added the tip strings and fixed the misalign in the bottom text. Thanks again! 😄

**Pull request merged**
